.gitignore

```diff
@@ -0,0 +1,6 @@
+
+node_modules
+build
+npm-debug.log
+.env
```

Comment:
do you know about the env file?
read this:
https://medium.com/@sujathamudadla1213/what-is-the-use-of-env-8d6b3eb94843#:~:text=env%20file%20is%20used%20in,them%20directly%20into%20your%20code.